ldap_fluff.gemspec

```
@@ -16,10 +16,10 @@ Gem::Specification.new do |s|
   s.email    = %w[jomara@redhat.com elobatocs@gmail.com pchalupa@redhat.com
                   komidore64@gmail.com mhulan@redhat.com dominic@cleal.org]
 
-  s.required_ruby_version = '>= 2.4.0'
+  s.required_ruby_version = '>= 2.7', '< 4'
 
-  s.add_dependency('activesupport')
-  s.add_dependency('net-ldap', '>= 0.11')
-  s.add_development_dependency('minitest')
-  s.add_development_dependency('rake')
+  s.add_dependency('activesupport', '~> 7.1', '>= 7.1.2')
```

This will severely limit us and break things. Foreman still runs with Rails 6.1 There's no reason to limit this so strictly.

While looking at this, I wondered why it even needs activesupport at all. There's this one subtle line:
https://github.com/theforeman/ldap_fluff/blob/00f35389d0c569d5cb3358cc4620fc51aa0db059/lib/ldap_fluff/config.rb#L32

That is the only reason it's needed. That was introduced in activesupport 3.1 but you can pick a later version. I'd now go with
```suggestion
  s.add_dependency('activesupport', '>= 6', '< 8')
```